docs/user-guide/pod-preset/index.md

```diff
@@ -0,0 +1,540 @@
+---
+assignees:
+- jessfraz
+title: Pod Preset
+---
+
+Objects of type `podpreset` allow for injecting certain information into pods
+at creation time such as secrets, volumes, volume mounts and environment
+variables.
+
+See [PodPreset proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/pod-preset.md) for more information.
+
+* TOC
+{:toc}
+
+## What is a Pod Preset?
+
+A _Pod Preset_ is an API resource that allows additional runtime requirements
```

Comment:
"an API resource that allows" -> "an API resource that you can use to inject additional runtime requirements into a Pod at creation time. You use label selectors to specify the Pods to which a given Pod Preset applies."